src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -1,20 +1,31 @@
 import * as _ from 'lodash';
 
-import { sbvrUtils, hooks } from '@balena/pinejs';
+import { hooks } from '@balena/pinejs';
 
-const updateLatestRelease = async (
-	id: number,
-	{ request, api }: hooks.HookArgs,
-) => {
-	// We only track builds that are successful
-	if (request.values.status !== 'success') {
+const updateLatestRelease = async ({ request, api }: hooks.HookArgs) => {
+	if (
+		!(
+			'status' in request.values ||
+			'is_passing_tests' in request.values ||
```

Comment:
You can change this to something like 
```
//This will go outside the function
const releaseStateFields = ['status', 'is_passing_tests', 'release_type', 'is_invalidated']

if(!releaseStateFields.some(field => field in request.values)) {
  return
}
```

Reply:
thanks, that's indeed a bit cleaner!